src/packages/excalidraw/README.md

```diff
@@ -270,3 +272,21 @@ This callback is triggered when the shareable-link button is clicked in the expo
 1. `exportedElements`: An array of [non deleted elements](https://github.com/excalidraw/excalidraw/blob/6e45cb95dbd7a8be1859c7055b06957298e3097c/src/element/types.ts#L76) which needs to be exported.
 2. `appState`: [AppState](https://github.com/excalidraw/excalidraw/blob/4c90ea5667d29effe8ec4a115e49efc7c340cdb3/src/types.ts#L33) of the scene.
 3. `canvas`: The `HTMLCanvasElement` of the scene.
+
+#### `langCode`
+
+Determines the language of the UI. It should be one of the [available language codes](https://github.com/excalidraw/excalidraw/blob/d337c8b15f6c1085287b12ecbe59c96e2c4e0ff4/src/i18n.ts#L14). Defaults to `en` (English).
+We also export default language and supported languages which you can import as shown below.
+
+```js
+import { defaultLang, languages } from "@excalidraw/excalidraw";
+```
+
+| name        | type                                                                                                                 |
+| ----------- | -------------------------------------------------------------------------------------------------------------------- |
+| defaultLang | string                                                                                                               |
+| languages   | [Language []](https://github.com/excalidraw/excalidraw/blob/c35d983fef8a83ba842dd892c0f461111a3e8589/src/i18n.ts#L9) |
```

Comment:
This shouldn't be a permalink lest we update the type in the future. Though I agree the link may become stale either way, I think it's safer if it links to a non-existent place rather than to an old schema.

Reply:
The reason for using the perm link was so it doesn't get invalidated soon due to any edits/formatting changes (For example a new line) as this way it will invalidate very soon. 
So instead of using non-perm links, we add a comment stating `update the readme if any change` for places where we are using perm links in the readme.
do you think that would help? 
Also, I am not sure but the using non-perm links might get invalidated once the pull request is branched as commits are squashed / branch is deleted.

Reply:
Comments won't work, and will get out of date themselves. Let's leave it for now, we'll figure it out later.